traffic_ops_ort/atstccfg/config/config.go

```diff
@@ -104,6 +105,7 @@ func GetCfg() (Cfg, error) {
 	disableProxyPtr := flag.BoolP("traffic-ops-disable-proxy", "p", false, "Whether to not use the Traffic Ops proxy specified in the GLOBAL Parameter tm.rev_proxy.url")
 	dirPtr := flag.StringP("dir", "D", "", "ATS config directory, used for config files without location parameters or with relative paths. May be blank. If blank and any required config file location parameter is missing or relative, will error.")
 	viaReleasePtr := flag.BoolP("via-string-release", "", false, "Whether to use the Release value from the RPM package as a replacement for the ATS version specified in the build that is returned in the Via and Server headers from ATS.")
+	dnsLocalBind := flag.BoolP("dns-local-bind", "", false, "Whether to use the server's Service Addresses to set the ATS DNS local bind address.")
```

Comment:
Bucking the trend to not name new new var dnsLocalBindPtr makes it look different from the others, when it's really not.

Reply:
Eh, I originally did the `viaReleasePtr` thing. I think that was a mistake now, it adds complexity for no value. You're right, consistency is good, they should really all be changed

Reply:
Changed